src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
are you sure this doesn't cause d14 dlls to overwrite d15 dlls? I added these so that d15 becomes the ones that left in d15 test directory.